doc/api/esm.md

```diff
@@ -657,21 +668,24 @@ export async function resolve(specifier, context, defaultResolve) {
 }
 ```
 
-#### `getFormat(url, context, defaultGetFormat)`
+#### `load(url, context, defaultLoad)`
 
-> Note: The loaders API is being redesigned. This hook may disappear or its
-> signature may change. Do not rely on the API described below.
+> Note: Previously the experimental version had 3 separate hooks, which have
+> been consolidated into this one.
```

Comment:
I would undo this change. The design will change further after this. We don’t want references to “the previous design” as if there were only two (the current and the former).